dbos/_dbos_config.py

```diff
@@ -124,9 +206,6 @@ def get_dbos_database_url(config_file_path: str = DBOS_CONFIG_PATH) -> str:
 
 def load_config(
```

Comment:
I've been thinking about this more&mdash`load_config` is in our public API and is actually used by our demo apps, which means we need to preserve its behavior, which means we need to call `process_config` from `load_config` and return the result of that.